spec.md

```diff
@@ -600,45 +616,37 @@ operation.
 
 ## Blocking Operations
 
-Service Brokers do not have to support concurrent requests that mutate the
-same resource.  If a Service Broker receives a request that it is not
-able to process due to other activity being done on that resource then the
-Service Broker MUST reject the request with a HTTP `422 Unprocessable
-Entity` error and the following body (see [Service Broker Errors](#service-broker-errors):
-
-```
-{
-  "error": "ConcurrencyError",
-  "description": "Another operation for this Service Instance is in progress."
-}
-```
+Service Brokers do not have to support concurrent requests that mutate the same
+resource. If a Service Broker receives a request that it is not able to process
+due to other activity being done on that resource then the Service Broker MUST
+reject the request with a HTTP `422 Unprocessable Entity` and a response body
+containing error code `"ConcurrencyError"` (see [Service Broker
+Errors](#service-broker-errors)).
 
 Note that per the [Orphans](#orphans) section, this error response does not
-cause orphan mitigation to be initiated. Therefore, Platforms receiving
-this error response SHOULD resend the request at a later time.
+cause orphan mitigation to be initiated. Therefore, Platforms receiving this
+error response SHOULD resend the request at a later time.
 
-Brokers MAY choose to treat the creation of a binding as a mutation of
-the corresponding Service Instance - it is an implementation choice. Doing
-so would cause Platforms to serialize multiple binding creation requests
-when they are directed at the same Service Instance if concurrent updates
-are not supported.
+Brokers MAY choose to treat the creation of a binding as a mutation of the
+corresponding Service Instance - it is an implementation choice. Doing so would
+cause Platforms to serialize multiple binding creation requests when they are
+directed at the same Service Instance if concurrent updates are not supported.
 
 ## Polling Last Operation
 
 When a Service Broker returns status code `202 Accepted` for
 [Provision](#provisioning), [Update](#updating-a-service-instance), or
 [Deprovision](#deprovisioning), the Platform will begin polling the
 `/v2/service_instances/:instance_id/last_operation` endpoint to obtain the
-state of the last requested operation. The Service Broker response MUST
-contain the field `state` and MAY contain the field `description`.
+state of the last requested operation. The Service Broker response MUST contain
+the field `state` and MAY contain the field `description`.
```

Comment:
Just curious - did you manually notice that "contain" could fit on the previous line or does your editor automatically move it up for you?  Unless there's some other change in there I'm not seeing.

Reply:
Manually noticed it :)